app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncMessage.java

```diff
@@ -0,0 +1,92 @@
+package org.openmrs.eip.app.management.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sender_sync_message")
+public class SenderSyncMessage extends AbstractEntity {
+
+    private static final long serialVersionUID = 1L;
+
+    @NotNull
+    @Column(name = "table_name", length = 100, nullable = false, updatable = false)
+    private String tableName;
+
+    @NotNull
+    @Column(length = 255, nullable = false, updatable = false)
+    private String identifier;
+
+    @NotNull
+    @Column(length = 1, nullable = false, updatable = false)
+    private String operation;
+
+    @NotNull
+    @Column(name = "message_uuid", length = 38, nullable = false, unique = true, updatable = false)
```

Comment:
length should be 32

Reply:
The ticket specifies 32 as the length, but the UUID has 32 chars + 4 dashes, resulting in 36 chars, so 32 is not suffice, unless we remove the dashes...
Then I took a look to the existing source code to see how the UUID is stored on other entities, and I've found that we are using 38 chars, and I did the same. Some examples bellow:

- `org.openmrs.eip.app.management.entity.BaseSyncRequest`
```
	@Column(name = "request_uuid", nullable = false, unique = true, updatable = false, length = 38)
	private String requestUuid;
```

- `org.openmrs.eip.app.management.entity.DebeziumEvent`
```
	@AttributeOverride(name = "requestUuid", column = @Column(name = "request_uuid", unique = true, updatable = false, length = 38))
	private Event event;
```

So, should we generate a UUID without the dashes to be able to store on a 32 char column?